src/server/controllers/data.ts

```diff
@@ -34,4 +35,19 @@ const cards: CardProps[] = [
   },
 ]
 
-export default cards
+export const users: UserProps[] = [
```

Comment:
Wäre auch geeignet das aus einem JSON file auszulesen, finde aber in dem scope auch okay 👍